package.json

```diff
@@ -70,6 +68,7 @@
     "@types/webpack-env": "^1.16.0",
     "@typescript-eslint/parser": "^4.26.0",
     "concurrently": "^6.2.0",
+    "dayjs": "^1.11.7",
```

Comment:
I agree with Ian on this. It should be moved back to the `dependences`

Reply:
you mean `peer dependencies`

Reply:
My bad 😅  Yes `peerDependencies`